src/components/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+
+export default class Header extends React.Component<void, void> {
+
+  render(): JSX.Element {
+    return(
+      <nav className="navbar navbar-default">
+        <span className="navbar-brand">Welcome to the Library Registry interface!</span>
```

Comment:
Semantically, some message like this should not be in the `nav` bar but could be an `h1` on the page. You'd have the main name of the app in the header/nav but not necessarily a welcome message there.

Reply:
It's just a placeholder; I figure the point of the header is that there will eventually be a logout button there.  (I'll change it anyway, though; might as well!)